backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -0,0 +1,10 @@
+[id="about-installing-oadp"]
+= About installing OADP
+include::modules/common-attributes.adoc[]
+:context: about-installing-oadp
+
+toc::[]
+
+You install the Openshift API for Data Protection (OADP) by installing the OADP Operator and then installing the Data Protection Application.
+
+You must have S3-compatible object storage for storing backups and a cloud provider that supports snapshots. You specify the backup location and snapshot location in the `DataProtectionApplication` custom resource manifest.
```

Comment:
>You specify the backup location and snapshot location in the `DataProtectionApplication` custom resource manifest.
`snapshot location` is not required.  to back up persistent volume data, you have 3 options:
1. configure snapshot location by using cloud provider storage snapshot feature
2. enable CSI snapshot, for example, we can install OCS 4 CSI snapshot backup
3. Enable restic to do PV bakcup

Reply:
I thought they had not yet decided whether we are documenting Restic for GA.

Reply:
yeah, we can confirm with dev or PM about Restic for GA